.gitignore

```diff
@@ -19,6 +19,9 @@ bld/
 [Bb]in/
 [Oo]bj/
 
+# Visual Studio cache/options directory
```

Comment:
That's nice but it has nothing to do with Timeout Config & Exception, feel free to create a separate PR.